README.md

```diff
@@ -1,2 +1,72 @@
 # napari-plugin-devtools
-A repo with tools and services for napari plugin developers
+
+[![License](https://img.shields.io/pypi/l/napari-plugin-devtools.svg?color=green)](https://github.com/napari/napari-plugin-devtools/raw/main/LICENSE)
+[![PyPI](https://img.shields.io/pypi/v/napari-plugin-devtools.svg?color=green)](https://pypi.org/project/napari-plugin-devtools)
+[![Python Version](https://img.shields.io/pypi/pyversions/napari-demo.svg?color=green)](https://python.org)
+[![tests](https://github.com/napari/napari-plugin-devtools/workflows/CI/badge.svg)](https://github.com/napari/napari-plugin-devtools/actions)
+
+A repo with tools and services for [napari plugin](https://napari.org/docs/dev/plugins/index.html) developers
+
+## Validation tool
+The validation tools provides more transparency and confidence for 
+plugin developers through automated tests, these checks verify that 
+the plugin is available for users of napari to install, and would 
+register entry points with napari. 
+
+There are two parts to the validation 
+tool, one part being CLI and another part being pytest fixture, 
+
+### CLI Usage
```

Comment:
lots of jargon.  write out "command line interface" and "continuous integration" in at least one place in the doc